### PR TITLE
Stop using the terms "global default", "default project", and "default environment".

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1199" title="DX-1199" target="_blank">StoryDX-1199</a>  We have a better way of explaining the "global default" to users
  </td></table>
  <br />

### CODE REVIEW

**Review by rawktron — APPROVED** [review]
Looks good. Noticed a bunch of places we'll have to update when we promote state use to stable.

**Comment by mitchell-as** [comment]
@MDrakos Could I get a stamp from you on this? Pete approved, but this PR needs approval from someone with repo write access.

**Comment by mitchell-as** [comment]
@MDrakos Wait, sorry I read it wrong. I think this PR is good to go. Sorry about that.

**Pull request merged**
